osmosis/assetlist.json

```diff
@@ -1302,6 +1305,7 @@
           }
         }
       ],
+      "coingecko_id": "autism",
```

Comment:
Please remove this cgid (we only want them defined in the assetlist corresponding to the (usually just one) chain they are originally issued).

Reply:
fixed